src/mcp/server.ts

```
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+process.env.DOTENV_CONFIG_QUIET = 'true';
```

ESLint

[error] Replace `'true'` with `"true"`

(prettier/prettier)